test/docker-compose-mysql.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   mysql:
-    image: mysql:8.0.18
+    image: mariadb:bionic
```

Comment:
`bionic` is name of the Ubuntu base image. `10.4` is the tag for the current stable release.

Reply:
It is also applied to their 10.4 tag https://hub.docker.com/_/mariadb but we can make it more explicit.

Reply:
The equivalent to `postgres:12-alpine` would be `mariadb:10.4-ubuntu`, but since this tag is not available I though you mean `mariadb:10.4`. It's usually not necessary to worry about the base image version.